test_examples/fast/SharedExampes_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_helper'
+require 'minitest/spec'
+
+class Minitest::SharedExamples < Module
+  include Minitest::Spec::DSL
+end
+
+SharedExampleSpec = Minitest::SharedExamples.new do
```

Comment:
You defined shared example here. Could you add at the bottom a new describe and include there shared example so we can see if they can be executed and recorded by knapsack (Travis will do that https://github.com/ArturT/knapsack/blob/master/.travis.yml#L62 ).